### ISSUE TITLE
maybe there exist some problems with glyphs U+06C1“ہ”~U+06C3“ۃ”, and others…

### ISSUE THREAD

**Issue by MY1L** [notofonts/arabic#211]
### Font 
 Noto Sans Arabic, and Naskh…

### Where the font came from, and when
   Site: https://notofonts.github.io/fonts/NotoSansArabic/unhinted/otf/NotoSansArabic-Regular.otf
   Date: 2023年10月30日

### Font Version
 2.011

#### Observed results

U+06C1“ہ”~U+06C3“ۃ”:
![image](https://github.com/notofonts/arabic/assets/58043328/1eff5365-6a14-4ea5-b2b2-9befefd1a169)
I think glyphs U+06C1 as U+FBA7“ﮧ”, and U+06C2,U+06C3 :
![image](https://github.com/notofonts/arabic/assets/58043328/45ff5568-85e3-4acb-bc6f-1a74a2bad456) (also NotoSansArabic-Regular v2.011)

#### Expected results
https://www.unicode.org/charts/PDF/U0600.pdf
![image](https://github.com/notofonts/arabic/assets/58043328/44717c9b-b961-411f-9e6a-8e77e7233e28)
https://www.unicode.org/charts/PDF/UFB50.pdf
![image](https://github.com/notofonts/arabic/assets/58043328/5b3ff343-6807-4967-b469-7886786bfa7b)

**Comment by MY1L** [comment]
### Issue 2 (unimportant)
maybe it's just a difference in arab calligraphy?

#### Observed results
NotoSansArabic-Regular v2.011 U+10EFD:
![image](https://github.com/notofonts/arabic/assets/58043328/58268368-b0c1-4aec-9da3-e9ab1ed4e58e)

#### Expected results
![image](https://github.com/notofonts/arabic/assets/58043328/864357b0-d36a-4834-838d-de3562a2bbaa)（[MY1L：Last v8](https://github.com/MY1L/Unicode/releases/tag/Last8)）
https://www.unicode.org/charts/PDF/U08A0.pdf
https://www.unicode.org/charts/PDF/U10EC0.pdf
![image](https://github.com/notofonts/arabic/assets/58043328/245e6d4f-2cb5-4a2f-8fad-d2f141734511)

**Comment by khaledhosny** [comment]
As for the first issue, that is not an issue. The Unicode charts use the final form of U+06C1 to make it standout since its main difference from other heh-like letters is in the final form.

The other issue I think is a Unicode inconsistency, it is the same word and the proper Arabic spelling of the word would be with two dots. The [proposal for U+10EFD](https://www.unicode.org/L2/L2021/21133-quranic-marks.pdf) shows only one source, so I can’t tell if the omission of the dots is intentional or just an oddity, so I’d rather keep it consistent unless someone has a evidence that the presence of the dots here is an issue,

**Comment by MY1L** [comment]
### Issue 3 (unimportant)
maybe these glyphs Weight mismatch?
uni0898
uni0899
uni089A
uni089B
u10EFE
u10EFF

#### Observed results
NotoSansArabic-Regular v2.011:
![image](https://github.com/notofonts/arabic/assets/58043328/36ede565-fded-4ff7-b03a-562b371049b2)

#### Expected results
between Bold and SemiBold🤔
NotoSansArabic-Bold：
![image](https://github.com/notofonts/arabic/assets/58043328/c9db751a-27c7-45d0-952c-051efbb23e16)

NotoSansArabic-SemiBold：
![image](https://github.com/notofonts/arabic/assets/58043328/ed45289c-23a2-433d-a806-caa6e9839cb0)

**Comment by simoncozens** [comment]
Agree, something's definitely gone wrong with weights here, compare U+10EFD and U+10EFE

Light: <img width="258" alt="Screenshot 2023-11-04 at 17 05 17" src="https://github.com/notofonts/arabic/assets/106728/8ab7b4bd-ee6f-457f-8a77-5a9ba6d44ef6">
Regular: <img width="270" alt="Screenshot 2023-11-04 at 17 05 33" src="https://github.com/notofonts/arabic/assets/106728/547f8d07-99f3-4887-87cc-f896b36c71d5">
Semibold: <img width="279" alt="Screenshot 2023-11-04 at 17 05 50" src="https://github.com/notofonts/arabic/assets/106728/5eb29373-bd95-4bc4-ac05-7d25a7c3cf6c">
Bold: <img width="286" alt="Screenshot 2023-11-04 at 17 06 07" src="https://github.com/notofonts/arabic/assets/106728/adca1370-e9c3-4642-9af6-7b0ca2e611e3">

**Comment by khaledhosny** [comment]
Some of the glyphs are optically corrected, others (new ones) are not.

**Comment by khaledhosny** [comment]
Smart components would have been helpful here, if they weren’t PITA to setup in glyphs.
